### PR TITLE
Documentation - SDK Middleware Http

### PR DESCRIPTION
**Pull request by ajimae**
#### Summary

- Add information for `abortController | getAbortController`
- Add usage examples for abortController and getAbortController
- Add a bit of context of their main differences

#### Description

Expand on the already existing documentation on `abortController`, add usage examples and point out the difference between `abortController` and `getAbortController`

#### Todo

- Tests
  - [ ] Unit
  - [ ] Integration
  - [ ] Acceptance
- [x] Documentation
- [x] `Type` label for the PR

### CODE REVIEW

File: docs/sdk/api/sdkMiddlewareHttp.md [hunk 1]
```diff
@@ -79,6 +79,36 @@ const client = createClient({
...
+// Use default options
+const httpMiddleware = createHttpMiddleware({
+  host: testHost,
+  timeout: 1000, // time out after 1s
+  fetch,
+  getAbortController: () => new AbortController(),
+})
+```
+
+The is intentional as to ensure a new instance of the AbortController is always created and is independent of each other. Unlike the former (abortController) which only creates a single abortController instance for the middleware, in this very case, if a single request times out, it will propagate to all other http requests that is using the `AbortController` instance. This is useful when a bunch of sent out requests needs to timeout if at least one within the bunch times out.
```
Comment (barbara79) [inline docs/sdk/api/sdkMiddlewareHttp.md:110]: I guess you meant to write` js` and not 'is'  ;) so please write it as a code block

**Comment by barbara79** [comment]
It looks good to me 🥇  just a small comment on my side ;)

**Review by barbara79 — APPROVED** [review]

File: docs/sdk/api/sdkMiddlewareHttp.md (hunk 1 above)
Comment (ajimae) [reply docs/sdk/api/sdkMiddlewareHttp.md:110]: Thanks @barbara79 do you mean the part that says:

The is intentional as to ensure... ?

Comment (barbara79) [reply docs/sdk/api/sdkMiddlewareHttp.md]: yes

Comment (barbara79) [reply docs/sdk/api/sdkMiddlewareHttp.md]: I just saw your change now it looks good :D

**Review by markus-azer — APPROVED** [review]
Thanks @ajimae , looks pretty good, just one comment, why there is changes in `yarn.lock`?

**Comment by ajimae** [comment]
> Thanks @ajimae , looks pretty good, just one comment, why there is changes in `yarn.lock`?

I ran `yarn install` when I checkout of master branch.

I will revert on changes in the `yarn lock` accordingly.

**Pull request merged**